about.qmd

```diff
@@ -84,7 +84,7 @@ Tyreesha has a background in privacy and compliance. In her role she supports da
 :::: {layout-ncol=2}
 ::: {#first-column }
 
-![Gina Conseca Crescioni - Data Governance Specialist](img/team/gina-fc.jpg){width=80%}
+![Gina Fonseca - Data Governance Specialist](img/team/gina-fc.jpg){width=80%}
```

Comment:
@seankross She asked for this change I assume?

Reply:
yes

Reply:
👍🏽